scripts/geojson/IFeatureUpdateStrategy.ts

```diff
@@ -0,0 +1,7 @@
+import { GeoJSONFeature, ClusterProps } from "./GeoJSONProps";
+import { Layer } from "leaflet";
+
+export interface IFeatureUpdateStrategy {
```

Comment:
I don't quite like the name of this interface, maybe something like IFeatureRenderer could be more appropriate since there are add and update methods.